addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -40,14 +47,74 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
     quickEditExclusion: [
         '[href]',
     ],
-
+    dynamicPlaceholderOptions: {
+        readonly: false,
+        needDefaultValue: true,
+        cancelOnEscape: true,
+        chainedTitle: true,
+        filter: (model) => !["one2many", "boolean", "many2many"].includes(model.type)
+    },
     custom_events: {
         wysiwyg_focus: '_onWysiwygFocus',
         wysiwyg_blur: '_onWysiwygBlur',
         wysiwyg_change: '_onChange',
         wysiwyg_attachment: '_onAttachmentChange',
     },
 
+    /**
+     * @override
+     */
+    init: function () {
+        this._super(...arguments);
+        // TODO this break the mass_mailing editor tour ( the content of the field html is reset )
+        //      why was this code here in the first place ?
+        // if (this.nodeOptions && this.nodeOptions.dynamic_placeholder) {
+        //     // When the dynamic placeholder is active,
+        //     // the recordData need to be updated when `mailing_model_real` change
+        //     this.resetOnAnyFieldChange = true;
+        // }
+    },
+
+    openDynamicPlaceholder: async function (baseModel, chain = []) {
```

Comment:
Wait what ? There is the same function above. Is this an override ? This function is missing doc with at least an `@override` mention.

Reply:
Indeed, I'll add some documentation for this function